### PR TITLE
Adding recent works  html and css file

### PR DESCRIPTION
**Pull request by Aseadd**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -218,7 +218,7 @@ h4 {
   flex-grow: 0;
 }
 
-.card-work-row1 {
+.grid-work-row1 {
```
Comment (rloterh) [inline styles.css:221]: - [ ] We actually want you to use `display:grid` for the main work section. Kindly ensure to improve upon this aspect.

File: index.html [hunk 2]
```diff
@@ -11,69 +11,338 @@
...
+                            </ul>
+                        </div>
+                        <div class="icons">
+                            <div class="icon-languages">
+                                <div class="ellipse">
+                                    <div class="icon-langguage">
+                                        <div class="frame">
+                                            <img src="Lang.png" alt="language icon"/>
+                                        </div>
+                                    </div>
```
Comment (rloterh) [inline index.html]: - [ ] To enforce proper use of semantics kindly ensure the icon is wrapped with the `a` tag, as this serves as link.

File: index.html [hunk 3]
```diff
@@ -11,69 +11,338 @@
...
+                                       <li class="tag2"><h6 class="tag-name2">CSS </h6></li>
+                                        <li class="tag3"><h6 class="tag-name3">JavaScript</h6></li>
+                                        <li class="tag4"><h6 class="tag-name4">html</h6></li>
+                                </ul>
+                            </div>
+                        </div>
+
+                        <div class="actions">
+                            <div class="btn">
+                                <h6 class="btn-text">See Project</h6>
```
Comment (rloterh) [inline index.html]: To enforce proper use of semantics kindly ensure that **all** _buttons_ are wrapped with the `a` tag, as this serves as link.

File: index.html [hunk 4]
```diff
@@ -11,69 +11,338 @@
 </head>
 
 <body>
```
Comment (rloterh) [inline index.html:13]: - [ ] Your design appears aligned to the _left_. Kindly ensure that the contents of your page is **properly centered**, as seen in the [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

**Your design:**
image


**The actual design:**
image

Comment (rloterh) [inline index.html:13]: - [ ] OPTIONAL
A long scrollable empty space is showing at the end of your page. Kindly endeavor to rid the blank empty space.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Addis, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Properly implemented HTML semantics


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

You can also consider:
N/A
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Aseadd) [reply styles.css:221]: Changed to display: grid

**Comment by Aseadd** [comment]
@rloterh  here are my corrections

**Comment by Aseadd** [comment]
Dear Robert , I have made the changes. Please check it out. the deadline is
approaching and I need to make it for the next curriculum.
*Best Regards,*
*Addis Tsega (MSc)*
*Lecturer, Department of Software Engineering, Haramaya University*
*Contact: +251913409680*
*Email: ***@***.*** ***@***.***>*
*P.O.BOX: 138, Dire Dawa, Ethiopia*

File: index.html [hunk 5]
```diff
@@ -11,69 +11,339 @@
...
+                </p>
+                <p>below my contact</p>
+            </div>
+            <div class="social-media">
+                <ul class="social-nav">
+                    <li><a href="http://linkedin.com/AddisTsega"><i class="fa fa-linkedin"></i></a></li>
+                    <li><a href="http://linkedin.com/AdaTsega"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="http://linkedin.com/Aseadd"><i class="fa fa-github"></i></a></li>
+                    <li><a href="http://linkedin.com/Addis-Tsega"><i class="fa fa-facebook"></i></a></li>
+                </ul>
```
Comment (1GORDON) [inline index.html:44]: Nice UI @Aseadd. 🤝

- [ ] [Optional] It is better if you make the project similar to what is provided in the Figma design. Kindly change the color of your social media icons.
- Your design
Yours

File: index.html [hunk 6]
```diff
@@ -11,69 +11,339 @@
...
+            <div class="supporting-text">
+                <p>I develop websites and other software products with high quality.
+                    I have a series of experiences in building software for different customers. If you have any
+                    project, please contact me.
+                </p>
+                <p>below my contact</p>
+            </div>
+            <div class="social-media">
+                <ul class="social-nav">
+                    <li><a href="http://linkedin.com/AddisTsega"><i class="fa fa-linkedin"></i></a></li>
```
Comment (1GORDON) [inline index.html:40]: Great work @Aseadd 🤝

- [ ] [Optional] You may please try to add that arrow in your design and space your elements according to the Figma design
- Figma design
Figmas

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Aseadd ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it :ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Good feature branch name ✔️ 
- Good commit messages  ✔️
- Nice UI ✔️

## Optional suggestions

- [ ] [Optional] You may please always add a description of your project. 
description

- [ ] [Optional] You may also remove the second author from your readme file.

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**